### DISCUSSION TITLE
Why create a release for 2.289.3 when 2.290.1 is already out?

### DISCUSSION THREAD

**Discussion by uwehdaub** [actions/runner#1837] [Q&A] [answered]
Why create a release for 2.289.3 when 2.290.1 is already out?

**Comment by TingluoHuang** [comment, marked as answer]
we noticed the runner package has some outdated files when investigating https://github.com/actions/runner/issues/1812.
`2.289.3` is created so we can patch the old GHES release with it, a newer version `2.290.2` will also be out soon after testing and fixing the issue.
